### DISCUSSION TITLE
I don't recieve an email even after 10 minutes of waiting

### DISCUSSION THREAD

**Discussion by AmineDev07** [stoatchat/discussions#390] [General]
I don't recieve an email after i create an account and wait for 10 minutes i  don't know why?

# Issue Fixed - note from maintainer
There was a bug introduced on the 21st which prevented sign ups, we just deployed a fix.

**Comment by sussycatgirl** [comment]
Have you checked the spam folder?

**Reply by AmineDev07** [reply]
i checked everything literally even socials section and ads section

`can i hand you email in the mean time until this bug is fixed cuz i really want to test the app`

**Reply by insertish** [reply]
See edited parent post.

**Comment by laikahaskell** [comment]
I have also tried two different email addresses and got nothing, not even in spam. It seems that it is broken for the time being. I have heard similar things from other friends trying to sign up in the past 24 hour, apparently because the email server is swamped with requests.

**Reply by insertish** [reply]
> apparently because the email server is swamped with requests.

It's actually the complete opposite and the mail server wasn't even getting any emails to send out.

**Comment by 4444dogs** [comment]
Same - This really sucks
